File: data structures and algorithms/java/05_001.java
import java.util.*;

public class Main {
  public static void main(String[] args) {
     // Create a mutable ArrayList directly
     List<Integer> numbers = new ArrayList<>(
        Arrays.asList(1, 2, 3, 4, 5));
     
     // Dynamic operations on the same list
     numbers.add(6);      // Adds to end
     numbers.add(0, 0);   // Adds at beginning
     
     System.out.println(numbers); // [0, 1, 2, 3, 4, 5, 6]
  }
}

File: data structures and algorithms/cs/05_001.cs
using System;
using System.Collections.Generic;

public class MainClass
{
    public static void Main(string[] args)
    {
        // Create a mutable List directly
        List<int> numbers = new List<int> { 1, 2, 3, 4, 5 };

        // Dynamic operations on the same list
        numbers.Add(6);      // Adds to end
        numbers.Insert(0, 0);   // Adds at beginning

        Console.WriteLine(string.Join(", ", numbers)); // [0, 1, 2, 3, 4, 5, 6]
    }
}